src/tscpp/util/unit_tests/test_IntrusiveDList.cc

```diff
@@ -286,9 +297,10 @@ TEST_CASE("IntrusiveDList", "[libts][IntrusiveDList]")
 
   PrivateThingList priv_list;
   for (int i = 1; i <= 23; ++i) {
-    std::string name;
-    ts::bwprint(name, "Item {}", i);
-    priv_list.append(new PrivateThing(name));
+    char buff[64];
+    //    ts::bwprint(name, "Item {}", i);
```

Comment:
Remove commented out code?

Reply:
No, unfortunately. Because I'm bringing this back in pieces, rather than one massive PR, I need to break some internal dependencies. That's what the commented code is. I plan to restore it when I bring in the pieces on which it depends.

Reply:
ok. that makes more sense.